codex-rs/core/src/skills/loader.rs

```diff
@@ -88,86 +87,81 @@ where
         .skills
         .retain(|skill| seen.insert(skill.name.clone()));
 
-    outcome
-        .skills
-        .sort_by(|a, b| a.name.cmp(&b.name).then_with(|| a.path.cmp(&b.path)));
+    fn scope_rank(scope: SkillScope) -> u8 {
```

Comment:
If this internal only, it's fine, but if you want some breathing room, consider at least `i16`?